src/gpt-auto-generator/gpt-auto-generator.c

```diff
@@ -53,7 +53,8 @@ static int open_parent_block_device(dev_t devnum, int *ret_fd) {
 
         r = sd_device_new_from_devnum(&d, 'b', devnum);
         if (r < 0)
-                return log_debug_errno(r, "Failed to open device: %m");
+                return log_debug_errno(r, "Failed to open device "DEVNUM_FORMAT_STR": %m",
```

Comment:
maybe "open" is misleading actually, and "device" is imprecise.

maybe say "Failed to allocate device object for block device…"

Reply:
(i mean, if you touch this, let's fix his properly)